### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by YaraslauVashkevich**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
```
Comment (IliaPotapovich) [inline .idea/compiler.xml:7]: .idea shouldn't exist in git repository. Please create .gitignore file (read how it works) and remove .idea from repository

File: git/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (IliaPotapovich) [inline git/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml:1]: The same for git package and target. There are shouldn't exist in git repository. Please create .gitignore file (read how it works) and remove .idea from the repository

File: .idea/compiler.xml (hunk 1 above)
Comment (YaraslauVashkevich) [reply .idea/compiler.xml]: File .gitignore file create. Folder .idea from repository remove.

File: git/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml (hunk 2 above)
Comment (YaraslauVashkevich) [reply git/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml]: File .gitignore file create. Folder .git and target from repository remove.

File: logging/pom.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        <artifactId>upskill-lab</artifactId>
+        <groupId>com.epam.upskill</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>logging</artifactId>
+
+    <name>logging</name>
+    <!-- FIXME change it to the project's website -->
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply logging/pom.xml]: Useless comments remove.

File: logging/src/main/java/com/epam/upskill/Main.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import org.apache.logging.log4j.Logger;
+
+import javax.swing.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Main {
+    static Logger logger = LogManager.getLogger();
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply logging/src/main/java/com/epam/upskill/Main.java]: Field is private.

File: logging/src/test/java/com/epam/upskill/MainTest.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package com.epam.upskill;
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply logging/src/test/java/com/epam/upskill/MainTest.java]: Useless test class remove.

File: logging/target/classes/log4j2.xml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply logging/target/classes/log4j2.xml]: File .gitignore file create. Folder .idea, .git and target from repository remove.

File: junit/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml [hunk 7]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply junit/target/surefire-reports/TEST-com.epam.upskill.MainTest.xml]: File .gitignore file create. Folder .idea, .git and target from repository remove.

File: logs/log4j2.log [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+26.01.2021 12:28:02 [main]  INFO  upskill.Main : 42 - The configuration file is read.
+26.01.2021 12:28:29 [main]  INFO  upskill.Main : 51 - Directory selected.
+26.01.2021 12:28:29 [main]  INFO  upskill.Main : 65 - Renaming completed.
+26.01.2021 12:28:29 [main]  INFO  upskill.Main : 68 - The application is finished.
+26.01.2021 12:32:08 [main]  INFO  upskill.Main : 16 - The application is running.
+26.01.2021 12:32:16 [main]  INFO  upskill.Main : 24 - The configuration file is selected.
+26.01.2021 12:32:16 [main]  INFO  upskill.Main : 42 - The configuration file is read.
+26.01.2021 12:32:23 [main]  INFO  upskill.Main : 51 - Directory selected.
+26.01.2021 12:32:23 [main]  INFO  upskill.Main : 65 - Renaming completed.
+26.01.2021 12:32:23 [main]  INFO  upskill.Main : 68 - The application is finished.
```
[the remark this answers is not in the document]
Comment (YaraslauVashkevich) [reply logs/log4j2.log:18]: File .gitignore file create. Folder .idea, .git and target from repository remove.

**Review by IliaPotapovich — APPROVED** [review]

**Pull request merged**